### PR TITLE
Fix NetBSD build

### PR DESCRIPTION
**Pull request by 0-wiz-0**

### CODE REVIEW

File: src/layout-util.cc [hunk 1]
```diff
@@ -24,6 +24,7 @@
 #include <dirent.h>
 #include <sys/wait.h>
 #include <unistd.h>
+#include <sys/wait.h>
```
Comment (qarkai) [inline src/layout-util.cc:27]: The same include is on the line 25.

File: src/misc.cc [hunk 2]
```diff
@@ -34,6 +34,7 @@
 #include <memory>
 
 #include <langinfo.h>
+#include <locale.h>
```
Comment (qarkai) [inline src/misc.cc:37]: Already mentioned on line 29 as `<clocale>`.

**Review by qarkai — COMMENTED** [review]
@0-wiz-0 Have you tried build with the latest commits?

**Comment by 0-wiz-0** [comment]
No, I just built the latest release.
Please make a newer one then :)

**Pull request closed without merging**